src/Utilities/Compiler/Options/OptionKind.cs

```diff
@@ -3,7 +3,7 @@
 namespace Analyzer.Utilities
 {
     /// <summary>
-    /// Kind of option to fetch from <see cref="AnalyzerOptionsExtensions"/>.
+    /// Kind of option to fetch from <see cref="ICategorizedAnalyzerConfigOptions"/>.
```

Comment:
`cref="AnalyzerOptionsExtensions"` generates a compiler warning since it's conditionally compiled. But `ICategorizedAnalyzerConfigOptions` is still accurate too (it's the high level part that uses OptionKind)